local-setup-data/config.local.1.json

```diff
@@ -5,7 +5,7 @@
   "domain": "localhost:8080",
   "TLS": false,
   "dkgContractAddress": "0xB83ab42A05252a0eFe5678188B8516F531CB5Bb2",
-  "blockchainRPCURL": "https://rpc-mumbai.maticvigil.com/v1/bcce899b0024136d1f13572c49ba4c45b6164ce9",
+  "blockchainRPCURL": "https://polygon-mumbai.g.alchemy.com/v2/SywXkHRx93NFQmpnfzsOcu8M3HK7rK2r",
```

Comment:
Polygon mumbai contracts are still working for us?

Reply:
no it isnt, guess its not updated just yet.